.travis.yml

```diff
@@ -20,15 +20,18 @@ env:
   matrix:
     - COMPONENT=unit
     - COMPONENT=firebase
-      # TODO(#2738): Re-enable these when we find a fix.
-      #    - COMPONENT=instrumentation ANDROID_TARGET=16
-      #    - COMPONENT=instrumentation ANDROID_TARGET=17
-      #    - COMPONENT=instrumentation ANDROID_TARGET=18
+    - COMPONENT=instrumentation ANDROID_TARGET=16
+    - COMPONENT=instrumentation ANDROID_TARGET=17
+    - COMPONENT=instrumentation ANDROID_TARGET=18
     - COMPONENT=instrumentation ANDROID_TARGET=19
     - COMPONENT=instrumentation ANDROID_TARGET=21
     - COMPONENT=instrumentation ANDROID_TARGET=22
     - COMPONENT=samples
 
+before_install:
```

Comment:
You can probably just do this in the travis-create-emulator script, that way you do'nt have to add the sdkmanager update line and you'll avoid installing these on the matrices where they're not used (unit, samples, firebase)